feature(ft-sdk): document change for single tester (#62)
fivetran/fivetran_sdk: tools/connector-tester/README.md

The tester supports the following optional CLI arguments to alter its default behavior. You can append these options to the end of the docker run command provided in step 2 of [How To Run](https://github.com/fivetran/fivetran_sdk/blob/main/tools/connector-tester/README.md#how-to-run) section above.

> #### --port

Reviewer: same

Author: added back port cli with modified description

The tester supports the following optional CLI arguments to alter its default behavior. You can append these options to the end of the docker run command provided in step 2 of [How To Run](https://github.com/fivetran/fivetran_sdk/blob/main/tools/connector-tester/README.md#how-to-run) section above.

#### --port

> This option tells the tester on which port it should run.

Reviewer 2: ```suggestion
This option defines the port the tester should run on.
```
